moveit_setup_assistant/src/widgets/end_effectors_widget.h

```diff
@@ -71,7 +71,7 @@ class EndEffectorsWidget : public SetupScreenWidget
 
   EndEffectorsWidget(QWidget* parent, moveit_setup_assistant::MoveItConfigDataPtr config_data);
 
-  /// Recieved when this widget is chosen from the navigation menu
+  /// Received when this widget is chosen from the navigation menu
```

Comment:
remove the third slash

Reply:
I just automatically replaced the typo. However, I can do the slashes too.

Reply:
No, stop. Three slashed mark this function to be documented with doxygen.